gadget.Dockerfile

```diff
@@ -36,7 +36,7 @@ FROM docker.io/kinvolk/traceloop:202109280218336f4efc as traceloop
 # https://github.com/kinvolk/bcc/commit/1d78d60b2319aa36e6e8c120966d9502ba0a0b75
 # See BCC section in docs/CONTRIBUTING.md for further details.
 
-FROM quay.io/kinvolk/bcc:1d78d60b2319aa36e6e8c120966d9502ba0a0b75-focal-release
+FROM quay.io/kinvolk/bcc:aaef5774fd4e145af8fe97c59ee2c39938bc44d1-focal-release
```

Comment:
I created a new PR to avoid this confusion. 

> BTW, can I ask if there is a specific motivation to update BCC or it is just to keep it update?

The commit wasn't include in the image tag before, so it was not immediate to know with one was used.